coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -48,6 +48,8 @@ def print_beautified_diff(difflines, printer):
                           color='red')
             current_line_subtracted += 1
         else:
+            printer.print(format_line(line[1:],
+                                      real_nr=current_line_added))
```

Comment:
I doubt whether we should only use `current_line_added`. The way GitHub uses to print the line number is to provide both `current_line_subtracted` and `current_line_added` for unchanged lines.

Reply:
<img width="1267" alt="Screen Shot 2019-03-09 at 1 42 29 PM" src="https://user-images.githubusercontent.com/30733262/54068295-59fcfb00-4271-11e9-90ca-8f2af3984d6e.png">

@li-boxuan  Is this okay ? I think this is close to how GitHub prints.

Reply:
This looks great to me 👍. @Makman2 what's your view on it?